lib/editor/util/validation.js

```diff
@@ -175,21 +182,29 @@ export function validate (
       }
     case 'LATITUDE':
       const isNotLat = value > 90 || value < -90
-      if (isRequiredButEmpty || isNotLat) {
-        if (isNotLat) {
-          reason = 'Field must be valid latitude.'
+      if (isNotLat) {
+        reason = 'Field must be valid latitude.'
+        return {field: name, invalid: isOptionalAndEmpty || isNotLat, reason}
+      }
+      if (entity && entity.location_type >= 2) {
+        if (isOptionalAndEmpty) {
+          reason = '1 Latitude and Longitude are required for your current location type'
+          return {field: name, invalid: isOptionalAndEmpty || isNotLat, reason}
         }
-        return {field: name, invalid: isRequiredButEmpty || isNotLat, reason}
       } else {
         return false
       }
     case 'LONGITUDE':
       const isNotLng = value > 180 || value < -180
-      if (isRequiredButEmpty || isNotLng) {
-        if (isNotLng) {
-          reason = 'Field must be valid longitude.'
+      if (isNotLng) {
```

Comment:
^^ I am not sure that the above was resolved. Is the longitude check required when stop type is 'platform', for instance?

Reply:
Same goes for the latitude check.

Reply:
For the particular code above the Lat/LNG is required if the location_type is either 0, 1 or 2.  It is not based on Platform type.  I have it so that the isNotLng or isNotLat is checked regardless of the location type.